view/sass/content.scss

```diff
@@ -0,0 +1,98 @@
+* {
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+}
+
+.content-section {
+  font-family: 'Fjalla One', sans-serif;
+  height: 100vh;
+  width: 100vw;
+}
+
+.content-header {
+  background: #009688;
+  color: #fff;
+  display: flex;
+  font-size: 1.5em;
+  justify-content: space-between;
+  height: 10%;
+  width: 100%;
+}
+
+.logo, .search {
+  align-items: center;
+  display: flex;
+  height: auto;
+  justify-content: center;
+  width: 30%;
+
+  form {
+    align-items: center;
+    display: flex;
+    height: 100%;
+    width: 90%;
+  }
+
+  input{
+    border-radius: 4px;
+    border: none;
+    height: 50%;
+    text-align: center;
+    width: 100%;
+  }
+}
+
+.container{
+  height: 90%;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.video-container{
```

Comment:
Acredito que o nome desta classe não está muito claro

Reply:
@mvoliveira is on fire!

Reply:
ok